Core/Src/main.c

```diff
@@ -711,23 +714,18 @@ static void MX_GPIO_Init(void)
 void StartDefaultTask(void *argument)
 {
   /* USER CODE BEGIN 5 */
-  int i = 0;
 
   /* Infinite loop */
   for(;;) {
 
     /* Pet watchdog */
     HAL_IWDG_Refresh(&hiwdg);
     /* Toggle LED at certain frequency */
-    HAL_GPIO_TogglePin(GPIOC, GPIO_PIN_8); // I am not using MPU interface because I'm lazy
-
-    if (i % 2 == 1)
-     serial_print(".\r\n");
-    else
-     serial_print("..\r\n");
-
-    i++;
+    printf(".\r\n..\r\n");
+    HAL_GPIO_TogglePin(GPIOC, GPIO_PIN_8);
 
+    /* Send NERO state data continuously */
+    send_nero_msg();

```

Comment:
I kinda feel wierd including this here but idk if there is a better way to do it.

Reply:
agree but think its fine for now lets explore more options later

Reply:
Peyton wanted state messages to be sent periodically bring it up with him